sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -1488,6 +1577,13 @@ class MapboxMap internal constructor(
       Map.clearData(resourceOptions, callback)
     }
 
+    @VisibleForTesting(otherwise = PRIVATE)
+    internal val DEFAULT_STYLE_TRANSITION = TransitionOptions.Builder()
```

Comment:
I'll create an issue in gl-native later so that this default object could be returned from gl-native _before_ we actually load a style (for now it returns empty object).

Reply:
not relevant anymore